src/pages/mesh/advanced/caching/index.md

```diff
@@ -75,11 +94,27 @@ When the response includes cache-control values, only the [most restrictive valu
 
 <InlineAlert variant="info" slots="text"/>
 
-POST requests are not supported, and GET requests are limited to 2,048 characters.
+GET requests are limited to 2,048 characters.
+
+###  Response headers
```

Comment:
Response headers is a super important aspect of caching. I imagine a logical ordering of sections like:
1. Overview (using cache-control headers)
2. Resolving conflicts between sources
3. Overriding cache-control using mesh-level responseConfig
4. Verifying the caching behavior using response headers


That said, an overview can help re-emphasize the role of response headers in this feature.

e.g. "Caching is controlled by the directives within the `cache-control` response headers returned by the sources. At least one source included in the GraphQL query **_must_** return a cache-control header for the entire request to be considered for caching. The user does not need to explicitly include the `cache-control` header this in the source's `responseConfig`."